src/App.js

```diff
@@ -1,15 +1,18 @@
 import "./App.css";
-import { Home, SingleVideo, Liked, WatchLater, History } from "./pages";
+import {RequiresAuth} from "./components";
+import { Home, SingleVideo, Liked, WatchLater, History, Login, SignUp } from "./pages";
 import { Routes, Route } from "react-router-dom";
 
 export default function App() {
```

Comment:
You can change this to use es6 syntax of arrow functions

Reply:
will look into this, thanks ravi